framework/core/hpp_image.cpp

```diff
@@ -81,7 +81,7 @@ HPPImage::HPPImage(HPPDevice              &device,
 HPPImage::HPPImage(HPPDevice &device, HPPImageBuilder const &builder) :
     HPPAllocated{builder.alloc_create_info, nullptr, &device}, create_info{builder.create_info}
 {
-	get_handle()           = create_image(create_info.operator const VkImageCreateInfo &());
+	get_handle()           = create_image(create_info.operator VkImageCreateInfo &());
```

Comment:
moving the `pNext` manipulation into the builder logic would also remove the need for this change.

Reply:
Addressed in my last commit